### ISSUE TITLE
Lets make it more then a emulator.

### ISSUE THREAD

**Issue by onlysumitg** [tn5250j/tn5250j#21]
[This is not a issue. But I am not sure how to contact you guys so writing here]

This is a very good emulator. I was using it from quite a long time but I never saw its code. Recently I got a task which was not possible in RPG/DSPF so I went to  tn5250j code to solve the issue. Task was to replicate autocomplete field functionality for Green screen. With some modifications in tn5250j I actually created a autocomplete field in DSPF.(Link https://www.linkedin.com/groups/59314/59314-6184903711488299012) 

 I am not a master of JAVA but I understand JAVA. I must say tn5250j is high potential project and It can be more then a emulator for AS400. What I am trying to say we can built tn5250j as a framework for AS400 with functionality that is not supported by native interface.

I may be wrong but I think Screen Change Listener need to be improve to track specific events. Right now for a single event this listener is fired multiple time and Its not easy to identify the required event and I think strokenizer can be improved to queue up virtual key events,

I would like to contribute in this project if required. Is any technical documents available for tn5250j which can describe the flow of code?

**Comment by nitram509** [comment]
@onlysumitg 

Hi Sumit,

thank you very much for the flowers.
I appreciate you've already created an autocomplete field in DSPF.
Contributions are always welcome. Feel free to fork this project and create pull request.
So that other users of TN5250j will also benefit from your extension.

Regards
Martin

**Comment by SaschaS93** [comment]
@nitram509

I agree with @onlysumitg absolutely. A great project! But why doesn't anything happen anymore? Not even small pull requests are accepted anymore...
